visualizer.html

```diff
@@ -555,6 +556,24 @@
 	}
 	fileSave(commentFreeKnitout, "knitoutFile.k");
 }
+
+function saveJavascript(){
+	fileSaveJavaScript(editor.getValue(), "knitoutFile.js");
+
+}
+function fileSaveJavaScript(sourceText, fileIdentity){
```

Comment:
The `fileSaveJavasScript` function appears to be identical to `fileSave` -- could it be eliminated and `saveJavascript` modified to call `fileSave(editor.getValue(), "knitoutFile.js")`?

Reply:
should be resolved! I removed the redundant function